### PR TITLE
Docs for manual linking in PushNotificationsIOS

### LINKED ISSUE #3831: [PushNotification] RCTPushNotificationManager.h not found
I followed everything here (https://facebook.github.io/react-native/docs/pushnotificationios.html
) to use PushNotification on iOS, but when I try to build the app, I get the following error. 

`RCTPushNotificationManager.h file not found`

I even added `RCTPushNotification.xcodeproj` to Libraries but it still get the same error.

### PR DESCRIPTION
**Pull request by magus**
close #3831

### CODE REVIEW

**Comment by magus** [comment]
@satya164

**Comment by satya164** [comment]
Thanks.

**Comment by satya164** [comment]
@facebook-github-bot shipit

File: Libraries/PushNotificationIOS/PushNotificationIOS.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ var NOTIF_REGISTER_EVENT = 'remoteNotificationsRegistered';
  * To get up and running, [configure your notifications with Apple](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AddingCapabilities/AddingCapabilities.html#//apple_ref/doc/uid/TP40012582-CH26-SW6)
  * and your server-side system. To get an idea, [this is the Parse guide](https://parse.com/tutorials/ios-push-notifications).
  *
- * To enable support for `notification` and `register` events you need to augment your AppDelegate.
+ * [Manually link](https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking) the PushNotificationIOS library
+ *
+ * - Be sure to add the following to your `Header Search Paths`:
+ * ```$(SRCROOT)/../node_modules/react-native/Libraries/PushNotificationIOS```
```
Comment (brentvatne) [inline Libraries/PushNotificationIOS/PushNotificationIOS.js]: You only need a single tick here, `$(SRCROOT)/../node_modules/react-native/Libraries/PushNotificationIOS`

Comment (magus) [inline Libraries/PushNotificationIOS/PushNotificationIOS.js]: Good call, changed it

**Comment by magus** [comment]
Gah, deleted source branch

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by satya164** [comment]
Let's try again.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**